.github/workflows/check-dependencies.yml

```diff
@@ -1,10 +1,6 @@
 name: "3rd-party check"
 
 on:
-  push:
-    branches:
-      - main
-      - 'release-*'
```

Comment:
keep consistent with the `hugegraph`?
https://github.com/apache/incubator-hugegraph/blob/d4b95ca32371dfdee60d448fa3e5ab38fc4e288e/.github/workflows/check-dependencies.yml#L6

Reply:
Thanks, i'll take a look, just ref: https://github.com/apache/incubator-hugegraph-toolchain/pull/516 before.

Reply:
> Thanks, i'll take a look, just ref: [apache/incubator-hugegraph-toolchain#516](https://github.com/apache/incubator-hugegraph-toolchain/pull/516) before.

The reason is the running ci **context is a little different**, we need `Check License Header` in `push` action but don't need with `Dependency Review`, so consider:
1. just remove `push` action like this
2. separate them for different **trigger logic** (may need check the action doc) OR just divide into 2 CI files